source/documentation/deploying_services/s3.md

```diff
@@ -1,18 +1,23 @@
 # Amazon S3
 
-Amazon S3 provides object storage through a web service interface.
+Amazon S3 provides data object storage through a web service interface.
+
+The Amazon S3 backing service is in private beta and may not be suitable for all services. To request access and discuss your services needs, contact the GOV.UK PaaS team at [gov-uk-paas-support@digital.cabinet-office.gov.uk](mailto:gov-uk-paas-support@digital.cabinet-office.gov.uk).
 
 <h2 id="set-up-the-service">Set up the service</h2>
 
 ### Provision an Amazon Web Service S3 bucket
 
 Creating a service instance provisions an Amazon Web Service (AWS) Simple Storage Service (S3) bucket.
 
-The name of the service overall is `aws-s3-bucket`. There is one type of plan for the service, named `default`. This plan has unlimited data, storage, and resources in general
+The name of the service overall is `aws-s3-bucket`. There’s one type of plan available for the service named `default`. This plan has unlimited data, storage, and resources.
 
-#### Provision a private AWS S3 bucket
+S3 buckets can be either private or public:
+
+- private S3 buckets are only accessible using apps that you bind to that bucket
+- public S3 buckets are accessible to everyone through the internet
```

Comment:
Only last thought on this: the bucket itself is not accessible by everyone, only the individual objects themselves. ie:

If my bucket is called `test-bucket-123`, and I have uploaded a file to it called `this_is_a_funky_file.txt`, I could then create a link to `https://test-bucket-123.s3.amazonaws.com/this_is_a_funky_file.txt`, and anyone on the internet would be able to click the link and view the file.

HOWEVER: they would not be able to list the contents of the bucket to find any other files - I could also have a file in the bucket, called `this_is_a_super_secret_file.pdf`, and nobody would know it was there unless I told them the URL to it.

The long and short of it is:

* In a private bucket, the files are only accessible by authorised users (in this case, apps / service keys bound to the cloudfoundry service)
* In a public bucket, files can only be written and listed by authorised users, but anyone can download them if provided the URL

> IMPORTANT: even though a random on the internet couldn't list the contents of the bucket, that doesn't mean they're safe at all. Someone could brute-force or guess filenames. Therefore, only files that are absolutely 100% fine for someone random on the internet to download and view should be stored in a public bucket.

Reply:
How much of that is actually useful / usable in the context of this documentation is unknown.

Reply:
@tnwhitwell thanks for this. I think the first bullet is ok given that we already say about bound apps and we aren't including anything about service keys in the content, but we could clarify the second bullet to the following:
`- files in public S3 buckets can be downloaded by anyone with the URL for those files`

Reply:
@AP-Hunt can you take a look?

Reply:
I'm fine with that @jonathanglassman. Let's :shipit:

Reply:
Thank you. I think we need to wait on merging until feature is live?